### PR TITLE
added override blur-up classes

### PR DESCRIPTION
**Pull request by otmusic**

### CODE REVIEW

File: plugins/blur-up/ls.blur-up.js [hunk 1]
```diff
@@ -18,6 +18,28 @@
...
+
+		var blurUpDefaults = {
+			blurUpClass: 'ls-blur-up-img',
+			blurUpLoadingClass: 'ls-blur-up-is-loading',
+			blurUpInviewClass: 'ls-inview',
+			blurUpLoadedClass: 'ls-blur-up-loaded',
+			blurUpLoadedOriginalClass: 'ls-original-loaded'
+		};
+
+		lazySizesCfg = window.lazySizesConfig || window.lazysizesConfig || {};
```
Comment (aFarkas) [inline plugins/blur-up/ls.blur-up.js:34]: We do not rely on the global `lazySizesConfig`. You can simply copy it from `lazySizes.cfg` + copy it at the start where you declare the variable -> `var lazySizesCfg  = lazySizes.cfg;`

Comment (otmusic) [reply plugins/blur-up/ls.blur-up.js]: @aFarkas done

**Comment by otmusic** [comment]
@aFarkas done

**Comment by aFarkas** [comment]
@otmusic Thank you very much.

**Comment by otmusic** [comment]
@aFarkas 
You are welcome. Thank you for this really awesome library!

**Pull request merged**